src/main/java/com/bitquest/bitquest/BitQuest.java

```diff
@@ -811,6 +811,8 @@ public boolean onCommand(CommandSender sender, Command cmd, String label, String
                     } catch (IOException e1) {
                         e1.printStackTrace();
                     }
+                    if (sendAmount >= 20000)
+                    {
                     try {
```

Comment:
I'm not sure if it will work or not -- I don't have time to test this myself -- but if you're contributing to an existing project you should make the style of your code match the existing codebase.  If nothing else, making sure you're indenting correctly will improve the readability and make your PR more likely to be accepted.  Explodi or someone else may take it as-is, but not making the person reviewing and accepting PRs do more work to clean up after you is a considerate thing to do.